flow-typed/submissions.js

```diff
@@ -1,13 +1,18 @@
 // @flow
+// @ts-nocheck
 
 declare type NewDraftSubmission = {
   submission: {
     +[key: string]: mixed,
   },
   definition: Form,
+  previousFormSubmissionApprovalId: number | void,
 }
 
-declare type NewFormSubmission = NewDraftSubmission & {
+declare type NewFormSubmission = $Diff<
+  NewDraftSubmission,
+  {| previousFormSubmissionApprovalId: number | void |}
+> & {
```

Comment:
This can be undone.

Reply:
do you mean `NewFormSubmission` can have `previousFormSubmissionApprovalId?: number` in it? I did it this way because i was sure you'd not want it in `NewFormSubmission`